pro/admin-tasks.md

```diff
@@ -0,0 +1,696 @@
+
+### Searching for a report
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff</span>
+
+
+Click on ‘admin’ in the top menu bar.
+
+There are a number of different options for using the Search Reports box:
+
+- Enter any word contained in the required report’s title or body
+- Search by the report-maker’s name (or part of the name), their email address or phone
+    number (where given).
+- Search by the report’s ID number if it’s known. Putting ‘id:’ directly before this number in
+    the search box will speed up the return of results.
+
+<img alt="There are a number of different options for using the Search Reports box" src="/assets/img/pro-user-guide/search-report.png" class="pro-screenshot" />
+
+The search will return a list of all reports that fulfil your parameters. Click on the report’s ID
+number to see it in a new browser window; or click on the word ‘Edit’ to gain access the tools that
+(depending on the staff user’s permissions) allow editing of reports, prioritisation, changing a
+report’s status, etc.
+
+</div>
+
+<div class="pro-task" markdown="1" id="search-user">
+
+### Searching for a user
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff</span>
+
+Click on ‘admin’ in the top menu bar.
+
+Enter any part of the user’s name or email address in the ‘Search users’ box.
+
+</div>
+
+<div class="pro-task" markdown="1" id="editing-users-information">
+
+### Editing a user’s information
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff; "Edit users' details/search for their reports" must be ticked.</span>
+
+Click on ‘Edit’ beside any user to view or change their account details.
+Or, click the user’s email address to view a list of all reports and updates made by that user.
+
+</div>
+
+<div class="pro-task" markdown="1" id="view-reports-non-staff-user">
+
+### Viewing reports as a non-staff user
+
+As a staff user, your view of a report includes moderation functions which ordinary users do not
+see. If you would like to view a report as a member of the public would see it, there are three
+options:
+
+- Log out of FixMyStreet, or
+- Start a ​[private browsing session​](https://www.howtogeek.com/269265/how-to-enable-private-browsing-on-any-web-browser/), or
+- View the page in a different browser (e.g. Chrome rather than Internet Explorer), in which you are
+not logged in to FixMyStreet.
+
+Any of these courses of action will allow you to view the site as an anonymous visitor, with no
+recognition of your staff role.
+
+</div>
+
+<div class="pro-task" markdown="1" id="create-reports-behalf-user">
+
+### Creating reports on behalf of another user
+
+<span class="pro-task__permissions">Permissions required: User must be marked as staff; ‘Create reports/updates on a user's behalf’ must be ticked. ‘Create reports/updates as anonymous user’ and ‘Create reports/updates as the council’ are also options.</span>
```

Comment:
Note you could just have `create as the council` perm and not 'on a user's behalf' if that was the only one wanted, does that make sense? Perhaps title should be "Creating reports on behalf of another user/ the council/ etc" and then this would say you only needed one or more of them.